openapi.yaml

```diff
@@ -299,6 +299,13 @@ paths:
           schema:
             type: string
           example: Passed
+        - name: status
+          in: query
+          description: |
+            Execution Status for the run. Commonly queried values: 'building','generating','running'
```

Comment:
Why not list all the statuses ?

Are the status strings being compared case-sensitively ? or insensitively ?